src/components/search/search-no-results.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const SearchNoResult = ({ query = false }) =>
```

Comment:
Would be better to use an empty string as default `({ query = '' })` so you keep the same type.
Then use `query.length > 0 && (`

Reply:
Got it, yeah, thanks!